lib/matplotlib/tests/test_legend.py

```diff
@@ -50,7 +51,7 @@ def test_legend_auto2():
     x = np.arange(100)
     b1 = ax.bar(x, x, align='edge', color='m')
     b2 = ax.bar(x, x[::-1], align='edge', color='g')
-    ax.legend([b1[0], b2[0]], ['up', 'down'], loc=0)
+    ax.legend([b1[0], b2[0]], ['up', 'down'], loc='best')
```

Comment:
`loc='best'` is default, so maybe could be omitted altogether?

Reply:
Yes. I've removed them.